app/views/users/_form_fields.html.erb

```diff
@@ -41,6 +41,12 @@
   <%= f.select :organisation_id, organisation_options(f), organisation_select_options, { class: "chosen-select form-control", 'data-module' => 'chosen' } %>
 </p>
 
+<% if policy(User).flag_2sv? %>
+  <p class="form-group">
+    <%= f.check_box :requires_2sv %> <%= f.label :requires_2sv, "Requires 2SV" %>
```

Comment:
I think the wording of this should be tweaked - for one thing we should say "2 step verification", not "2SV". @fofr do you have any thoughts?